triggers/tests/BUILD.bazel

```diff
@@ -153,6 +154,7 @@ da_scala_library(
             "//daml-lf/engine",
             "//daml-lf/interpreter",
             "//daml-lf/language",
+            "//daml-script/converter",
```

Comment:
Dependency required so that we may inspect the DAML script converter exception in testing